backend/src/main/java/pl/edu/agh/apdvbackend/utilities/JwtTokenUtilsImpl.java

```diff
@@ -0,0 +1,38 @@
+package pl.edu.agh.apdvbackend.utilities;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import java.util.Date;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import pl.edu.agh.apdvbackend.models.database.User;
+
+@Component
+@RequiredArgsConstructor
+public class JwtTokenUtilsImpl implements JwtTokenUtils {
+
+    private static final int ACCESS_TOKEN_EXPIRES_TIME = 10 * 60 * 1000;
+
+    private static final int REFRESH_TOKEN_EXPIRES_TIME = 30 * 60 * 1000;
```

Comment:
jakis powód dlaczego akurat taki czas?

Reply:
gdzieś widziałam że 30 minut, ale trochę może być to przesadą, idk

Reply:
Ja w kolejnym PR dałem chyba z 10h na access token i 72h na referesh token XD

Reply:
Imo mogą być dłuższe te czasy, zwłaszcza ten drugi token, żeby użytkownik nie musiał się co chwilę logować